### PR TITLE
[Docs] Update mmdet.md

### PR DESCRIPTION
**Pull request by Luo-Z13**
[Docs] Update mmdet doc

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: .github/workflows/prebuild.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
           cd pack
           python ../tools/package_tools/generate_build_config.py --backend 'ort' \
             --system linux --output config.yml --device cpu --build-sdk --build-sdk-monolithic \
-            --build-sdk-python --sdk-dynamic-net
+            --build-sdk-python --sdk-dynamic-net --onnxruntime-dir=$ONNXRUNTIME_GPU_DIR
```
Comment (RunningLeon) [inline .github/workflows/prebuild.yml:45]: @irexyc Hi, is this change OK?

File: docs/zh_cn/04-supported-codebases/mmdet.md [hunk 2]
```diff
@@ -210,7 +210,8 @@ cv2.imwrite('output_detection.png', img)
 |       [RepPoints](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/reppoints)       |    ObjectDetection    |      N      |    Y     |  N   |   ?   |    Y     |
 |            [DETR](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/detr)            |    ObjectDetection    |      Y      |    Y     |  N   |   ?   |    Y     |
 |       [CenterNet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/centernet)       |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
-| [Cascade Mask R-CNN](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/cascade_rcnn) | InstanceSegmentation  |      Y      |    N     |  N   |   N   |    Y     |
+|       [RTMDet](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)             |   Object Detection    |      Y      |    Y     |  N   |   ?   |    ?     |
```
Comment (RunningLeon) [inline docs/zh_cn/04-supported-codebases/mmdet.md:213]: Could you double the supported models with model configs in mmdet.yml, and update mmdet.yml as well.
https://github.com/open-mmlab/mmdeploy/blob/6c26e887d4300c3afb57a0900c6ad53798d9dfda/tests/regression/mmdet.yml#L168-L353

Comment (RunningLeon) [inline docs/zh_cn/04-supported-codebases/mmdet.md:213]: The english md should be updated as well: `docs/en/04-supported-codebases/mmdet.md`

File: .github/workflows/prebuild.yml (hunk 1 above)
Comment (Luo-Z13) [reply .github/workflows/prebuild.yml:45]: Sorry, I didn't find the right way to do it after I failed to submit a PR, so I may have modified this file by mistake

**Comment by RunningLeon** [comment]
pls fix lint error by `pip install pre-commit && pre-commit install && pre-commit run --all-files`

File: docs/zh_cn/04-supported-codebases/mmdet.md (hunk 2 above)
Comment (Luo-Z13) [reply docs/zh_cn/04-supported-codebases/mmdet.md:213]: Of course, I'll update as soon as possible

Comment (Luo-Z13) [reply docs/zh_cn/04-supported-codebases/mmdet.md:213]: I update English md in https://github.com/open-mmlab/mmdeploy/pull/1983, sry, as this is my first attempt at submitting a PR, I have split it into two separate PR submissions

Comment (RunningLeon) [inline docs/zh_cn/04-supported-codebases/mmdet.md:213]: may close PR https://github.com/open-mmlab/mmdeploy/pull/1983 and put all update into this PR.

Comment (Luo-Z13) [reply docs/zh_cn/04-supported-codebases/mmdet.md:213]: Okay, that's best.

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @Luo-Z13 ！We are grateful for your efforts in helping improve mmdeploy open-source project during your personal time. 

Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）

Thank you again for your contribution❤

File: .github/workflows/prebuild.yml (hunk 1 above)
Comment (irexyc) [reply .github/workflows/prebuild.yml:45]: Hi @Luo-Z13 , we don't need to add this args. There exists a env variable of `ONNXRUNTIME_DIR` and wil be used by default. You can add a commit to remove the change or use `git revert` to undo that commit.

Comment (Luo-Z13) [reply .github/workflows/prebuild.yml:45]: > Hi @Luo-Z13 , we don't need to add this arg. There exists a env variable of `ONNXRUNTIME_DIR` and wil be used by default. You can add a commit to remove the change or use `git revert` to undo that commit.

Okay, I will change it soon after I fix lint error and update mmdet.yml as well.

**Comment by RunningLeon** [comment]
@Luo-Z13 hi, could you update this PR?

**Comment by Luo-Z13** [comment]
> @Luo-Z13 hi, could you update this PR?

I will finish it within today

**Comment by Luo-Z13** [comment]
> @Luo-Z13 hi, could you update this PR?

I'm sorry to bother you, I meet some problem after git clone：the origin branch's name doesn't match to my fork repo's name, and I try to
 # 通过全量单元测试
pytest tests 

but failed because lack of dependencies, what should I update PR properly?

**Comment by RunningLeon** [comment]
@Luo-Z13 hi, could sync with `main` branch and fix conflicts?

**Comment by Luo-Z13** [comment]
> @Luo-Z13 hi, could sync with `main` branch and fix conflicts?

I sync with main and merge en/cn change of mmdet.md in https://github.com/open-mmlab/mmdeploy/pull/2033 .

**Comment by RunningLeon** [comment]
@

> > @Luo-Z13 hi, could sync with `main` branch and fix conflicts?
> 
> I sync with main and merge en/cn change of mmdet.md in #2033 .
@Luo-Z13 
We'd like to close this pr and move all update to #2033

**Pull request closed without merging**